### PR TITLE
fix: disable following on starknet spaces

### PR DESCRIPTION
**Pull request by wa0x6e**
### Summary

This PR will hide the space FOLLOW/UNFOLLOW button on starknet spaces.

### How to test

1. Go to /explore?p=snapshotx
2. Hover some spaces to show the FOLLOW/UNFOLLOW button
3. It should be visible only on evm spaces

### CODE REVIEW

File: apps/ui/src/components/ButtonFollow.vue [hunk 1]
```diff
@@ -1,5 +1,9 @@
 <script setup lang="ts">
 import { Space } from '@/types';
+import { METADATA } from '@/networks/starknet';
+
+// NOTE: Disabling on Starknet until we have support for starknet long addresses on sequencer
+const DISABLED_NETWORKS = Object.keys(METADATA);
```
Comment (Sekhmet) [inline apps/ui/src/components/ButtonFollow.vue:6]: It's a bit hacky to get list of starknet networks through this, can we instead make it a whitelist and use already defined constants `evmNetworks` and `offchainNetworks` as `SUPPORTED_NETWORKS`?

File: apps/ui/src/components/ButtonFollow.vue [hunk 2]
```diff
@@ -12,7 +16,9 @@ const followedSpacesStore = useFollowedSpacesStore();
 const { web3 } = useWeb3();
 
 const spaceFollowed = computed(() => followedSpacesStore.isFollowed(spaceIdComposite));
-const hidden = computed(() => web3.value?.type === 'argentx');
+const hidden = computed(
+  () => web3.value?.type === 'argentx' || SUPPORTED_NETWORKS.includes(props.space.network)
```
Comment (Sekhmet) [inline apps/ui/src/components/ButtonFollow.vue:20]: This should be reversed as it hides the button.

Comment (wa0x6e) [reply apps/ui/src/components/ButtonFollow.vue]: 😒

**Review by Sekhmet — APPROVED** [review]
utACK

**Comment by wa0x6e** [comment]
Closing, as not relevant anymore. Issue has been fixed on sequencer

**Pull request closed without merging**